test/transport_test.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const expect = require('chai').expect;
+const mockery = require('mockery');
+const sinon = require('sinon');
+const Events = require('events');
```

Comment:
Is there a reason this is uppercase? I also noticed this in Transport.js.

Reply:
its a class, so the notation should be camelcase.